frontend/public/locales/en/translation.json

```diff
@@ -78,7 +78,7 @@
   "confirmation": "Confirmation",
   "clickToSetLoopLocation": "Click on the map to set the approximate location of your loop. A dot will appear, you can manually change the radius in the field below.",
   "clickToChangeLoopLocation": "Click on the map to change the location of the loop",
-  "loopName": "Loop name",
+  "loopName": "Loop name (should include city/village name)",
```

Comment:
I think what they meant was that the loopname should be appended with the city/village name ;)

Reply:
They actually requested to add "(should include city/village name)" after the field name already in use!